src/APIFunctions/3DPrinting.js

```diff
@@ -37,10 +45,12 @@ export async function submit3DPrintRequest (printRequest) {
       email
     })
     .then(result => {
```

Comment:
this function doesn't have any response data that we care about, since we are just deleting something.

you can remove the `.then()` block all together and just keep the `.catch()` block